src/javascript/class-scanner.ts

```diff
@@ -211,6 +213,7 @@ export class ClassScanner implements JavaScriptScanner {
       astNode,
       properties,
       methods,
+      staticMethods,
```

Comment:
We don't want to emit a Class for something that's an element, so there's a bit of a hack in here to notice that something is actual an element and treat it as such. Otherwise we emit the Class.